src/distributions/adoptium/installer.ts

```diff
@@ -0,0 +1,151 @@
+import * as core from '@actions/core';
```

Comment:
directory name should change from src/distributions/adoptium to src/distributions/temurin